pandas/tests/indexes/test_base.py

```diff
@@ -1496,6 +1496,13 @@ def test_drop_tuple(self, values, to_drop):
             with pytest.raises(KeyError, match=msg):
                 removed.drop(drop_me)
 
+    def test_drop_with_duplicates_in_index(self):
```

Comment:
you could use the `index` fixture and do something like

```
def test_drop_with_duplicates(self, index):
    if len(index) == 0: return
    index = index.repeat(2)  #  <-- ensure duplicates
    res = index.drop(index[0])
```

Reply:
Better? Is there a way to construc expected without drop?